app/storage/src/main/java/com/fsck/k9/preferences/migrations/DefaultStorageMigrationsHelper.kt

```diff
@@ -0,0 +1,82 @@
+package com.fsck.k9.preferences.migrations
+
+import android.database.sqlite.SQLiteDatabase
+import androidx.core.content.contentValuesOf
+import app.k9mail.core.android.common.database.getStringOrThrow
+import app.k9mail.core.android.common.database.map
+import timber.log.Timber
+
+class DefaultStorageMigrationsHelper : StorageMigrationsHelper {
```

Comment:
I noticed that the plural form is used for `Migrations`. Also the package names are in plural form. I think we should use the singular form for package **_names_** and **_classes_**.

Reply:
I created a separate PR for this. See #7899.